flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
why not aarch64?

Reply:
Just because we couldn't test a build for aarch64. Is there a tool to emulate/test it?

Reply:
just build it. there is no reason to not to. It's python.